package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@grafana/lezer-logql",
-  "version": "0.0.15",
+  "version": "0.0.16",
```

Comment:
The package-lock.json should also be updated.

Reply:
Yes good catch, I've committed the fix.